### DISCUSSION TITLE
<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by NoltixOS** [community/community#68748] [Programming Help]
### Select Topic Area

Show & Tell

### Body

Bro, do you have that friend that say’s “Programming is stupid, What are you even doin’?” But he likes Robots, and he doesn’t understand’s that Without programming, Robots were stupid.

![IMG_2986](https://github.com/community/community/assets/127049334/cd25c652-accc-48ec-9f42-184a5f318781)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @XsoftBud ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions.

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
